interaction_lib.py

```
@@ -0,0 +1,307 @@
+        return {
+            'lr_stat': lr_stat,
+            'df': df,
+            'p_value': p_value,
+            'improvement': p_value < 0.05,
+            'aic_base': result_base.aic,
+            'aic_full': result_full.aic,
+            'bic_base': result_base.bic,
+            'bic_full': result_full.bic
+        }
+    
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)